scripts/ps-migrate-restore.sh

```diff
@@ -213,19 +201,15 @@ if [ "$DATA" ]; then
     unset PGUSER PGPASSWORD PGDATABASE
     printf "[SUCCESS]"
     echo ""
+fi
 
-    printf "Restoring postgresql data for pscheduler..."
-    export PGUSER=$(sed -n -e 's/.*user=\([^ ]*\).*/\1/p' /etc/pscheduler/database/database-dsn)
-    export PGPASSWORD=$(sed -n -e 's/.*password=\([^ ]*\).*/\1/p' /etc/pscheduler/database/database-dsn)
-    export PGDATABASE=$(sed -n -e 's/.*dbname=\([^ ]*\).*/\1/p' /etc/pscheduler/database/database-dsn)
-    psql --no-password < $TEMP_RST_DIR/$TEMP_BAK_NAME/postgresql_data/pscheduler.dump &>/dev/null
+#get pscheduler if exists
+if [ -e "$TEMP_RST_DIR/$TEMP_BAK_NAME/pscheduler" ]; then
```

Comment:
You may want to do a `which pscheduler` here to make sure it's installed as is done during backup.